src/main/java/nwoolcan/view/main/MainController.java

```diff
@@ -42,7 +42,7 @@ public void menuViewDashboardClick(final ActionEvent event) {
     public void menuViewWarehouseClick(final ActionEvent event) {
         getController().setBreweryName("ciccio");
         getController().setOwnerName("ciccia");
```

Comment:
why is this here?

Reply:
it is in master, it does not concern my pr. don't block the flow for things like these.

Reply:
@Zeegomo if you want to change that, open an issue and ask for explanations

Reply:
anyway, it should not be there

Reply:
> @Zeegomo if you want to change that, open an issue and ask for explanations

That's your part of view, it's pretty clear that this should not be on master